packages/common/src/TimeUtils.ts

```diff
@@ -11,22 +11,21 @@ export async function averageBlockTimeSeconds(chainId?: number): Promise<number>
   // TODO: Call an external API to get this data. Currently this value is a hard-coded estimate
   // based on the data from https://etherscan.io/chart/blocktime. ~13.5 seconds has been the average
   // since April 2016, although this value seems to spike periodically for a relatively short period of time.
-  const defaultBlockTimeSeconds = 13.5;
+  const defaultBlockTimeSeconds = 12;
   if (!defaultBlockTimeSeconds) {
     throw "Missing default block time value";
   }
 
   switch (chainId) {
-    // Block time is irrelevant on Arbitrum since one transaction ~= 1 block, so this time value is based on empirical
-    // observation as of January 4 2022 of Arbitrum block propogation.
+    // Source: https://polygonscan.com/chart/blocktime
+    case 10:
+      return 0.5; // 2 TPS, an underestimate
```

Comment:
Should we overestimate, so the blockfinder errs by going further back than it needs to?

Or do you mean 0.5 is an underestimate (meaning 2 TPS is higher than we would typically expect)?

Reply:
2 TPS is slower than we'd expect, is that what we desire?

Reply:
I would think we'd want to tend to overestimate if it's variable because that means the blockfinder would initially jump back farther on its first try to try to find a block _before_ the timestamp.

See [this initial loop](https://github.com/UMAprotocol/protocol/blob/25d22a9088a64da8f6783130c65d717a9104015d/packages/sdk/src/blockFinder.ts#L30-L44). It uses the estimated TPS to figure out how far to increment. If the TPS is an overestimate, it should (I think) increment _further_ than it should on the first try, immediately getting a lower bound on the block. Then, it begins its interpolation search (similar to binary search) to find a matching block. If the TPS is an underestimate, it will need to loop more times to find a lower bound before it begins the interpolation search. The only upside on the underestimate is that it will likely have a smaller window over which it needs to interpolation search. However, interpolation searches converge must faster than a "guess and check" loop backwards.

Reply:
ok so to over estimate the TPS, we should set a lower time per block, this will increment more blocks into the past. I'll set them to 1.5-2x the average current block time.